sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/models/_models.py

```diff
@@ -2356,19 +2356,19 @@ def __init__(
         super(Incident, self).__init__(**kwargs)
         self.metric_id = kwargs.get('metric_id', None)
         self.detection_configuration_id = kwargs.get('detection_configuration_id', None)
-        self.incident_id = kwargs.get('incident_id', None)
+        self.id = kwargs.get('id', None)
```

Comment:
should we include this rename in the Changelog?

Reply:
No. This is a bug rather than a renaming. We used id in docstring. :(